command/format/diff.go

```diff
@@ -1024,12 +1022,9 @@ func (p *blockBodyDiffPrinter) writeValueDiff(old, new cty.Value, indent int, pa
 			var changeShown bool
 
 			for i := 0; i < len(elemDiffs); i++ {
-				// In concise mode, push any no-op diff elements onto the stack
-				if p.concise {
-					for i < len(elemDiffs) && elemDiffs[i].Action == plans.NoOp {
-						suppressedElements = append(suppressedElements, elemDiffs[i])
-						i++
-					}
+				for i < len(elemDiffs) && elemDiffs[i].Action == plans.NoOp {
+					suppressedElements = append(suppressedElements, elemDiffs[i])
+					i++
```

Comment:
I expected this diff to follow the same `p.concise` 👉 `!p.verbose` change as the others. Any reason for removing the `if` statement? I'm concerned that it might result in surprisingly hidden elements in some edge case not covered by tests.

Reply:
`verbose` is only relevant when printing terraform state, which by definition won't have a `diff`. This particular function is only called when there are old and new values to compare. 

Having said that, I'll add it back in so we're consistent, just in case something gets refactored in the meantime to depend on this ~differently!